### DISCUSSION TITLE
My Samsung display does not seem to work. What can I do?

### DISCUSSION THREAD

**Discussion by cprada87** [MonitorControl/MonitorControl#584] [Q&A] [answered]
**Checklist**

- [x] I have searched for existing issues
- [x] I have looked through [the Monitor Troubleshooting wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [x] I have updated MonitorControl to the latest version

**Describe the issue**
I can't control any monitor status

**Screenshots**
<img width="256" alt="Captura de pantalla 2020-12-24 a las 16 10 08" src="https://user-images.githubusercontent.com/28503903/103096034-e2918a80-4602-11eb-8b9a-d948c777fa59.png">

<img width="762" alt="Captura de pantalla 2020-12-24 a las 16 22 55" src="https://user-images.githubusercontent.com/28503903/103096498-613af780-4604-11eb-81f3-85a09e139369.png">

**Environment Information (please complete the following information):**
 - macOS version: **Big Sur 11.1**
 - MonitorControl version: **2.1.0 (Build 721)**
 - Monitor(s): **Samsung U28570**
 - Mac model: **A2289**
 - Monitor Cable(s)/Connection(s): (e.g. DisplayPort) **USB-C to HDMI@60hz**
[https://www.amazon.es/gp/product/B08CSBPPR9/ref=ppx_yo_dt_b_asin_title_o07_s00?ie=UTF8&psc=1](url)

**Comment by cprada87** [comment]
any progress?

**Comment by BerkeAras** [comment]
Same issue :(

**Comment by BerkeAras** [comment]
_not working in 3.0.0-beta 2_

**Comment by waydabber** [comment]
Hi, can you please try the latest 3.0.0 RC1? Is it working properly with ddctl?

Also you could try setting DDC polling mode to 'None' under Displays/Advanced (you need to enable Advanced options in the General tab) and try if it works - as your display might provide erroneus max accepted values to MonitorControl and that's why you can't use the sliders.

<img width="803" alt="Screen Shot 2021-08-21 at 16 23 15" src="https://user-images.githubusercontent.com/37590873/130324814-ff10235f-50c8-42e3-a460-2389678a5ba6.png">

Thank you and please let me know if this helps!

**Comment by waydabber** [comment]
I'll close this issue for now but please feel free to comment if there is anything we can do. If the issue persists, we can reopen this issue! Thank you!

**Comment by cprada87** [comment]
the same @waydabber, still does not work, although now it recognizes the monitor model and allows me to move audio and brightness but without result.

**Comment by waydabber** [comment]
Hi @cprada87 - does it work properly via ddcctl or Lunar (@alin23)? The screenshot suggests so but it is still not clear if the display is actually controlled via an alternate app. Thank you!

**Comment by cprada87** [comment]
Hi @waydabber, neither of them :( BR

**Comment by alin23** [comment, marked as answer]
@cprada87 For brightness you could try disabling the features listed in the [Lunar FAQ](https://lunar.fyi/faq#brightness-not-changing) to see if DDC starts working.

<img width="1060" alt="Shot_Safari app_Lunar - Frequently Asked Questions_PqHww9q1" src="https://user-images.githubusercontent.com/3819725/133071173-1f2e7a44-c7a8-4cff-959c-f8950503653e.png">


For volume, try disabling the **Auto Mute** setting if it is enabled, and maybe post a screenshot with the sound settings of your monitor in case there are other features that could block volume changing through DDC.

**Reply by cprada87** [reply]
@alin23, @waydabber first, thank you for your reply and support!

I just checked all the options you mentioned (@alin23) in the Lunar FAQ and everything is off or not selected.

another question: if we get this done, could Lunar adjust both brightnesses (mbp & external monitor) using touchbar?

The only thing I can think of now is to change the cable to DisplayPort, currently I am using a hdmi@60hz direct to usb-c.

**Reply by alin23** [reply]
Yes, Lunar can do that already in Sync mode. Adjusting the Touchbar slider will change the Macbook builtin display and Lunar will kick in a few milliseconds later and sync that change to the external monitor.

I think @waydabber is already working on something similar for MonitorControl.

Trying DisplayPort is a very good idea, I've had many users report DisplayPort working when HDMI didn't.

**Reply by cprada87** [reply]
ok, regarding the feedback you receive from users, any particular model/brand of cable? BR

**Reply by alin23** [reply]
No, users never mentioned that unfortunately.

**Reply by helmarw** [reply]
i can confirm that, my 4x Samsung Monitors only the ones connected to displayport can be adjustet by MonitorControl
im using those cables https://www.amazon.de/dp/B08NJYCV8Y/ref=cm_sw_em_r_mt_dp_dl_E33HS7HFR6BP5C4ESZF2?_encoding=UTF8&psc=1

**Reply by alin23** [reply]
@helmarw very useful data point! I added that to the Lunar FAQ for other users having the same problem.

<img width="1056" alt="Shot_Safari app_Lunar - Frequently Asked Questions_h83zHTOo" src="https://user-images.githubusercontent.com/3819725/133077971-859bda13-a736-4dd6-87e2-34a3bb922743.png">

**Reply by waydabber** [reply]
> Yes, Lunar can do that already in Sync mode. Adjusting the Touchbar slider will change the Macbook builtin display and Lunar will kick in a few milliseconds later and sync that change to the external monitor.
> 
> I think @waydabber is already working on something similar for MonitorControl.

Yes, in the upcoming 3.1.0 and we plan to add this feature as well as it is [an old feature request](https://github.com/MonitorControl/MonitorControl/issues/27). Most of the stuff needed (the unified scale that allows syncing multiple display, polling the internal display brightness and even synchronization with the slider for the internal display) is already present and working in the latest code. But still, we are playing catch-up with Lunar in this regard so if you need this functionality right away in a mature app with a lot of great additional functionality, by all means try Lunar! 🥇

**Reply by cprada87** [reply]
Good news! A simple change of cable from hdmi@60hz to usb-c to use an aukey hub with a normal hdmi to hdmi cable...result works!

All this about cables is a world, and more now with HDCP content protection....one no longer knows what to buy and that is compatible to the maximum with the screen.

In the end we bought something relatively compatible with the qualities, but not with the functionality as you create, thank you very much for your time :D

**Reply by alin23** [reply]
So glad you finally got it fixed! Yep, it's hit and miss with cables nowadays. That's why I keep a drawer filled with dozens of similar looking cables with all kinds of connectors. One of them must work for what I need. 😅

**Reply by waydabber** [reply]
> Yes, Lunar can do that already in Sync mode. Adjusting the Touchbar slider will change the Macbook builtin display and Lunar will kick in a few milliseconds later and sync that change to the external monitor.
> 
> I think @waydabber is already working on something similar for MonitorControl.

Brightness change synchronization from Built-In and Apple displays to other displays is now added. This makes Touch Bar, Ambient light sensor, Control Center and System Preferences induced changes affect all displays. Synchronization is not direct, but only changes are replicated - the user can intervene and adjust at any time.

https://github.com/MonitorControl/MonitorControl/discussions/596
https://github.com/MonitorControl/MonitorControl/issues/27

**Comment by waydabber** [comment]
Oh boy I wish MonitorControl had such a nice FAQ! :)

I'll move this issue to a discussion for future reference (especially if the above helps).

**Reply by alin23** [reply]
Feel free to link to Lunar's FAQ whenever there's any relevant info there 😃 Copy pasting snippets from there is also ok as long as a link is included to the original page
